Automattic/wp-calypso
# State: Convert lib/billing-history-data to Redux

This PR converts the legacy `lib/billing-history-data` to Redux. It introduces Redux actions, reducer, selectors, schema and utilities for billing data. Also includes tests for those.

This PR is part of the billing history reduxification effort (#10554).

To test:

* Checkout this branch.
* Verify all tests pass: `npm run test-client client/state/billing-data/`.

/cc @gwwar
